gobblin-modules/gobblin-kafka-09/src/test/java/org/apache/gobblin/runtime/KafkaAvroJobStatusMonitorTest.java

```diff
@@ -118,6 +124,8 @@ public void setUp() throws Exception {
     builder = builder.withKafkaPusher(pusher).withKeys(Lists.newArrayList(TimingEvent.FlowEventConstants.FLOW_NAME_FIELD,
         TimingEvent.FlowEventConstants.FLOW_GROUP_FIELD, TimingEvent.FlowEventConstants.FLOW_EXECUTION_ID_FIELD));
     this.mysqlDagActionStore = mock(MysqlDagActionStore.class);
+    this.mockedGobblinServiceManager.when(() -> GobblinServiceManager.getClass(DagActionReminderScheduler.class)).thenReturn(mock(DagActionReminderScheduler.class));
+    this.mockedGobblinServiceManager.when(() -> GobblinServiceManager.getClass(DagActionStore.class)).thenReturn(mock(DagActionStore.class));
```

Comment:
this is a big PR!  I don't fully grasp what else changed that requires this mod.  please explain

Reply:
Because now we are calling static method `GobblinServiceManager.getClass` in our code, so we need to mock the static methods also in tests. mocking static methods is a little different from mocking instance methods and is done like this